apps/src/blockly/googleBlocklyWrapper.js

```diff
@@ -354,25 +354,8 @@ function initializeBlocklyWrapper(blocklyInstance) {
   ) {
     if (!this.unusedSvg_) {
       this.unusedSvg_ = new BlockSvgUnused(this, helpClickFunc);
+      this.unusedSvg_.render(this.svgGroup_, this.RTL);
     }
-    this.unusedSvg_.render(this.svgGroup_, this.RTL);
```

Comment:
Interesting that this ever-growing frame bug was already existing. It was never reported?

Reply:
Not to my knowledge! Most people probably dispose the frame before it becomes noticeable.